grasp_target_msg grasp_target
---
bool success
bool message
